_includes/analytics/google_analytics.html

```diff
@@ -1,10 +1,14 @@
+<!--
+  Analytics Account: "Austin Convention Center Department"
+  87703455-1 Profile: "ACC Alpha"
+-->
 <script>
   (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
   (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
   })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
-  ga('create', 'UA-83539778-3', 'auto');
+  ga('create', 'UA-87703455-1', 'auto');
```

Comment:
Ok. It still fetches ga.js on hard refresh, which is a bit annoying, but we can change it in a separate commit.